packages/mobile/src/utils/formatting.test.ts

```diff
@@ -30,22 +30,23 @@ describe('utils->formatting', () => {
     })
   })
 
-  describe('fees', () => {
-    it('rounds up the fees', () => {
-      const UNROUNDED_NUMBER = '0.00046'
-      const ROUNDED_NUMBER = '0.0005'
-      expect(getMoneyFeeyDisplayValueFromBigNum(new BigNumber(UNROUNDED_NUMBER))).toBe(
-        ROUNDED_NUMBER
-      )
+  describe('rounding', () => {
+    it('rounds up', () => {
+      const UNROUNDED_NUMBER1 = '0.50001'
+      const ROUNDED_NUMBER1 = '0.5001'
+      expect(roundUp(UNROUNDED_NUMBER1, 4).toString()).toBe(ROUNDED_NUMBER1)
+      const UNROUNDED_NUMBER2 = '0.599'
+      const ROUNDED_NUMBER2 = '0.6'
+      expect(roundUp(UNROUNDED_NUMBER2, 2).toString()).toBe(ROUNDED_NUMBER2)
```

Comment:
nit: maybe we shouldn't use intermediary variables here?

Reply:
Ok will fix